### PR TITLE
Add Access Delegation with OAuth2 Tutorial

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Reorder steps for deploying OIDC pickup samples
- Add 'configuring-access-delegation-with-oauth2' tutorial

### CODE REVIEW

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Configuring Access Delegation with OAuth 2.0
+
+### Introduction
+
+This tutorial demonstrates how WSO2 Identity Server (WSO2 IS) can be used as an OAuth 2.0 authorization server. It guides you through the OAuth 2.0 application configuration, deployment, and the usage scenario.
+
+In this sample scenario, a user tries to login to a vehicle booking web application called "pickup-dispatch" via a WSO2 Identity Server and recieves an access token to call the API to view the vehicle bookings.
```
Comment (gomathyk) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: via WSO2 Identity Server

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Configuring Access Delegation with OAuth 2.0
+
+### Introduction
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: ## Make changes in other places as well.

File: en/docs/learn/deploying-the-sample-app.md [hunk 3]
```diff
@@ -519,6 +503,28 @@ The next step is to configure the service provider.
...
+3. Next, copy the extracted and modified `pickup-dispatch` folder to the `<TOMCAT_HOME>/webapps` folder.
+
+4. Start the Tomcat server.
+
+To check the sample application, navigate to
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/pickup-dispatch/home.jsp         `
+on your browser.
+
+For example,
+`                     http://localhost:8080/pickup-dispatch/home.jsp                 .         `
```
Comment (gomathyk) [inline en/docs/learn/deploying-the-sample-app.md]: Is this localhost or localhost.com?

File: en/docs/learn/configuring-access-delegation-with-oauth2.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-access-delegation-with-oauth2.md]: Fixed with 
aeb5e16

File: en/docs/learn/deploying-the-sample-app.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/learn/deploying-the-sample-app.md]: Just 'localhost' works for the oidc pickup samples

File: en/docs/learn/configuring-access-delegation-with-oauth2.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-access-delegation-with-oauth2.md]: Fixed with 6a705ff23bc3f104517c7e2273c77e745fdadbda

Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: via WSO2 Identity Server

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+In this sample scenario, a user tries to login to a vehicle booking web application called "pickup-dispatch" via a WSO2 Identity Server and recieves an access token to call the API to view the vehicle bookings.
+
+![access-delegation-oauth-diagram](../assets/img/learn/access-delegation-oauth-diagram.png)
+
+### Setting up
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` and start the server by executing one of the following commands.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To start WSO2 Identity Server, navigate to the `<IS_HOME>/bin` directory via command prompt and execute the following command.

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    ``` java tab="Linux"
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: Linux/MacOS

Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: shell ?

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    sh wso2server.sh
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+	
+3. Follow the steps in
+   [deploying **pickup-dispatch** webapp](../../learn/deploying-the-sample-app/#deploying-the-pickup-dispatch-webapp)
+   to download, deploy and register **dispatch** sample.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To download, deploy, and register the **dispatch** sample, see [Deploying the Pickup Dispatch Webapp]().

//Fix the structure in other places.

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+4. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/backend-service-4.1.0.jar)
+   the `backend-service.jar` file from the latest release assets.
+
+5. Navigate to where the `backend-service.jar` file is located on a terminal window and run the following command to start the backend service.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To start the backend service, locate the `backend-service.jar` file in a command prompt and execute the following command.

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    ```
+    java -jar backend-service.jar -introspectionEnabled true
+    ```
+!!! Tip
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: !!! info

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    java -jar backend-service.jar -introspectionEnabled true
+    ```
+!!! Tip 
+    For more information about the backend service, see [Introduction to Backend Service](https://github.com/wso2/samples-is/tree/master/etc/backend-service).
+
+The setup is now complete and you can proceed to try out the scenario.
+
+### Try it out
+
+1. Run the application by visting the following URL: `http://localhost.com:8080/pickup-dispatch/index.jsp`
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To access the <aDD tHE nAME oF tHE aPPLICATION> application, visit `http://localhost.com:8080/pickup-dispatch/index.jsp`.

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+2. Login using the preferred credentials (e.g., admin/admin)
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: Log in

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+3. Next you have to provide consent to use your username.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: Provide consent 

user name

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+Backend calls are disabled by default. Enable it by selecting the **Backend** box in the drop down list as shown below.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To enable,

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+
+    ![enable-backend-calls-pickup-app](../assets/img/learn/enable-backend-calls-pickup-app.png)
+
+4. Add a new booking by selecting a driver from the drop down list and filling passenger details.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: To add a new booking,

File: en/docs/learn/configuring-access-delegation-with-oauth2.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+3. Next you have to provide consent to use your username.
+Backend calls are disabled by default. Enable it by selecting the **Backend** box in the drop down list as shown below.
+
+    ![enable-backend-calls-pickup-app](../assets/img/learn/enable-backend-calls-pickup-app.png)
+
+4. Add a new booking by selecting a driver from the drop down list and filling passenger details. 
+
+5. Click **Add**. On the backend-service log, you will notice that the request sent to `http://localhost:39090/bookings` contains an authorization header.
+
+6. You can click on the gear icon on the top-right corner to view the request and response.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-oauth2.md]: If this is an optional step, please use 'You may view the request and response by clicking on the ....', If not, just use 'To view the request and response, click the gear icon ...)

Comment (YvonneW) [reply en/docs/learn/configuring-access-delegation-with-oauth2.md]: Apply the same structure in other applicable places.

File: en/docs/learn/deploying-the-sample-app.md [hunk 15]
```diff
@@ -472,26 +472,10 @@ Follow the instructions below to download a sample from GitHub.
...
-
-For example,
-`                     http://localhost:8080/pickup-dispatch/home.jsp                 .         `
     
 ### Configuring the service provider
 
-The next step is to configure the service provider.
+The first step is to configure the service provider.
 
 1. Return to the WSO2 IS management console.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: To configure the service provider:

**Pull request merged**